### DISCUSSION TITLE
How to change clothes for characters in the world

### DISCUSSION THREAD

**Discussion by HotObsidian** [naverz/zepeto-studio-global#60] [Q&A]
I need to make a function that can control the player to change clothing, which interface in the character is to control the generation of clothing

**Comment by naverz-mkao** [comment]
Hi HotObsidian,

Currently we don't support clothing changes while in the world. There's good news though, that feature (we are calling in Mannequin) is on our Roadmap and we are working hard to make it available for all our creators!

You can see more details about the Mannequin feature in the link below!

https://github.com/orgs/naverz/projects/10

**Reply by HotObsidian** [reply]
Hello, what exactly is the Mannequin component you mentioned, I read the github document you posted and searched on the Internet, there is no relevant code and plug-in available for use

**Reply by ckdlskgkdrhd** [reply]
lol mkao literally said it's on their roadmap.
you obviously wouldn't find nothing.

**Reply by HotObsidian** [reply]
all right....

**Reply by naverz-mkao** [reply]
Hi HotObsidian,

Yes sorry if I was being unclear! The link that I gave you earlier is to our Roadmap GitHub page. There you will find features that we are planning on releasing in the future. These features are in various stages of completion. You can check back there anytime when you might be wondering what new features we are currently working on!

**Reply by HotObsidian** [reply]
OK i get it, thx~

**Reply by HotObsidian** [reply]
Hello, when I was doing the "Change clothes" function, I found that the clothes object did not have a Tag set in Unity. I couldn't locate the style of the clothes. Are there other tags representing the type of clothes in Unity?
